wp-content/themes/pub/wporg-learn-2020/template-parts/component-series-navigation.php

```diff
@@ -0,0 +1,46 @@
+<?php
+$series_term = wporg_workshop_series_get_term( $post );
+$previous    = wporg_workshop_series_get_adjacent( 'previous', $post );
+$next        = wporg_workshop_series_get_adjacent( 'next', $post );
+?>
+<nav class="workshop-series">
+	<h2>
+		<?php
+		printf(
+			wp_kses_post( __( 'Series: %s', 'wporg-learn' ) ),
+			sprintf(
+				'<a href="%1$s">%2$s</a>',
+				esc_html( get_term_link( $series_term ) ),
+				esc_html( $series_term->name )
+			)
+		);
+		?>
+	</h2>
+	<?php echo wp_kses_post( wpautop( term_description( $series_term->term_id ) ) ); ?>
+	<ul class="row video-grid">
+		<li class="col-6 video-grid_item">
+			<?php if ( $previous ) : ?>
```

Comment:
Can you move this above the `li`? Since the element has a `col-6` it will take up `50%` of the container even if it's empty.